packages/markdown-slate/package.json

```diff
@@ -24,7 +24,8 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/accordproject/slate-markdown-transform.git"
+    "url": "https://github.com/accordproject/slate-markdown-transform.git",
```

Comment:
I believe this is an outdated link.

Reply:
Sorry for that, I had changed the outdated link